az-core/src/main/java/azkaban/Constants.java

```diff
@@ -723,6 +723,12 @@ public static class ContainerizedDispatchManagerProperties {
     public static final String KUBERNETES_WATCH_ENABLED = KUBERNETES_WATCH_PREFIX + "enabled";
     public static final String KUBERNETES_WATCH_EVENT_CACHE_MAX_ENTRIES =
         KUBERNETES_WATCH_PREFIX + "cache.max.entries";
+    public static final String KUBERNETES_WATCH_CALL_TIMEOUT = KUBERNETES_WATCH_PREFIX + "call"
+        + ".timeout";
+    public static final String KUBERNETES_WATCH_CONNECT_TIMEOUT = KUBERNETES_WATCH_PREFIX +
```

Comment:
For all of these constants and literals please add the units in the suffix. For example
```
KUBERNETES_WATCH_CONNECT_TIMEOUT_SECONDS = KUBERNETES_WATCH_PREFIX +  "connect.timeout.seconds";
```